develop/scan-images/index.md

```diff
@@ -22,7 +22,7 @@ Docker and Snyk have partnered together to bring security natively into the deve
 
 Including the vulnerability scanning options into the Docker platform extends the existing, familiar process of vulnerability detection, and allows for remediation of vulnerabilities earlier in the development process. The process of simple and continuous checks, for example, by checking images in the background using [Snyk Advisor](https://snyk.io/advisor/docker){:target="_blank" rel="noopener" class="_"}, can lead to fewer vulnerabilities checked into Docker Hub. This can result in a shorter CI cycle and a more reliable deployment into production.
 
-![Developer's security journey](/images/dev-security-journey.png){:width="700px"}
+![Developer's security journey](images/dev-security-journey.png){:width="700px"}
```

Comment:
I'm confused; wow did this work? Because the old URL is a 404; https://docs.docker.com/images/images/docker-scan-cli.png, but the rendered page somehow uses the correct URL?

Reply:
I think it's not 404 because of the canonical url of this page:

```
<link rel="canonical" href="[https://docs.docker.com/develop/scan-images/](view-source:https://docs.docker.com/develop/scan-images/)" />
```

Reply:
Hmm.. not sure; if that was a `base` href, I could see that, but canonical shouldn't mean (afaik) anything other than "there's multiple pages with similar content; this URL is the canonical one".

No idea; odd